src/composables/transactionTokens.ts

```diff
@@ -10,7 +10,7 @@ import type {
 import {
   AETERNITY_SYMBOL,
   MAGNITUDE,
-  TX_FUNCTIONS,
+  DIRECTION,
```

Comment:
`TX_DIRECTION` would be even more appropriate.
`TX_DIRECTION.credit` instead of `received` and `TX_DIRECTION.debit` instead of `sent` is something you can consider too.